### ISSUE TITLE
Mouse controls questions

### ISSUE THREAD

**Issue by giagitom** [arose/ngl#352]
Hi, I have some questions about mouse controls.. 

First, I set a signal ```stage.signals.clicked.add(function(pikingProxy){...})```. The clicked event is fired even when I make mousedown+drag+mouseup. Is it possible to don't fire the event if drag occours? 

Second, the same event of the first question is fired with left, right and middle click.. Can I fire it only with left click?

Last, is it possible to fire an event on zoom or on mouse wheel?

**Comment by arose** [comment]
Hi, for more fine grained control use the `stage.mouseControls` instead of the signals for clicked/hovered events, see http://nglviewer.org/ngl/api/manual/usage/interaction-controls.html#Mouse

**Comment by arose** [comment]
@giagitom did the link help you? Do you have more questions?

**Comment by giagitom** [comment]
Hi! Yes thanks, I forgot to close :)

**Comment by giagitom** [comment]
One question.. I tryed to add this
```js
stage.mouseControls.add("clickPick",function(pikingProxy){console.log(pikingProxy)})
```
to get the atom pickd, but it does not work.. Am I missing something?

**Comment by ppillot** [comment]
You should try 
```javascript
stage.signals.clicked.add(function (pickingProxy) {console.log(pickingProxy)});
```

**Comment by ppillot** [comment]
Sorry, for the mouse observer, you should use this syntax instead
```javascript
stage.mouseObserver.signals.click.add(function (pickingProxy) {
 console.log(pickingProxy)
});
```

**Comment by giagitom** [comment]
Yes, it works and i was using it, but it take me to the original question, I don't want to pick after mousedown+drag.

**Comment by giagitom** [comment]
```js
stage.mouseObserver.signals.clicked.add(function (pickingProxy) {
 console.log(pickingProxy)
});
```
seems to work like this
```js
stage.signals.clicked.add(function(pikingProxy){...})
```

**Comment by arose** [comment]
no need to use mouseObserver directly, the following should work. Which version are you using @giagitom? Note that the signature is `(stage, pickingProxy)`. 

```javascript
stage.mouseControls.add('clickPick', function(stage, pickingProxy){
  console.log(stage, pickingProxy)
})
```

**Comment by giagitom** [comment]
Hi, after updating to the latest version "clickPick" works. But if i want to restrict the piking to only one mouse button (like "clickPick-middle" or "clickPick-left") it does not work anymore. 
My final goal is to be able to fire a center on atom with middle mouse button and to make an other task piking with left button.

**Comment by arose** [comment]
thanks @giagitom, fixed in 83a61a0e1c6504b4a96043085bcd247405f1bc65 will release later today

**Comment by arose** [comment]
released ngl@0.10.5-14 with the fix, thanks again

**Comment by giagitom** [comment]
Great! now it works. Thanks for the help!
